### DISCUSSION TITLE
is posible to add local theme to shiki on nuxt content v2 ?

### DISCUSSION THREAD

**Discussion by redfox-mx** [nuxt/content#1377] [Q&A]
I read some posts about creating a custom highlighter and adding to nuxt/content. Unfortunately, the highlight option was changed in version 2. Is it possible to implement some workaround to provide a custom theme to Shiki (as local theme, for example)?

**Comment by cossssmin** [comment]
Looks like it isn't possible: the [Shiki implementation](https://github.com/nuxt/content/blob/main/src/runtime/transformers/shiki/highlighter.ts) expects `theme` to be a string, which has to be one of the theme names available in Shiki.
